### PR TITLE
[Admin]  schedule delete

### PR DESCRIPTION
**Pull request by mvn-khanhnguyen-dn**
## Description
Create method destroy schedule in admin dasboard
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/Admin/ScheduleController.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            $bookingDetails = BookingDetail::with('ticket.schedule')
+                ->whereIn('ticket_id', $schedule->tickets()->get(['tickets.id']))
+                ->get();
+            foreach ($bookingDetails as $bookingDetail) {
+                $bookingDetail->delete();
+            }
+            $schedule->tickets()->delete();
+            $schedule->delete();
+            DB::commit();
+            return redirect()->back()->with('message', 'Delete successfully');
```
Comment (at-namtran) [inline app/Http/Controllers/Admin/ScheduleController.php]: locale chỗ này

File: app/Http/Controllers/Admin/ScheduleController.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        } catch (Exception $ex) {
+            DB::rollBack();
+            return redirect()->back()->with('message', 'Delete failed');
```
Comment (at-namtran) [inline app/Http/Controllers/Admin/ScheduleController.php]: locale chỗ này nữa

File: resources/views/admin/pages/schedules/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
+@extends('admin.layout.master')
+@section('title', __('schedule.admin.list.title'))
+@section('content')
+<div class="col-md-12">
+  <div class="tile">
+    <h3 class="tile-title">@lang('schedule.admin.list.title')</h3>
+    <div class="table-responsive">
+      @include('admin.layout.message')
+      @if (count($schedules))
```
Comment (at-namtran) [inline resources/views/admin/pages/schedules/index.blade.php]: chỗ này không cần if

Comment (at-khanhnguyen) [reply resources/views/admin/pages/schedules/index.blade.php]: @at-namtran  if chỗ ni để khi không có data thì cũng không hiện cái bảng.

**Pull request closed without merging**